prinz/src/main/scala/pl/touk/nussknacker/prinz/mlflow/repository/MLFRepositoryRestClient.scala

```diff
@@ -0,0 +1,41 @@
+package pl.touk.nussknacker.prinz.mlflow.repository
+
+import java.net.URL
+
+import pl.touk.nussknacker.prinz.mlflow.model.RegisteredModel
+import pl.touk.nussknacker.prinz.mlflow.MlflowConstants.{BASE_API_PATH, BASE_PREVIEW_API_PATH}
+import pl.touk.nussknacker.prinz.model.repository.ModelRepository
+import pl.touk.nussknacker.prinz.util.http.{RestClientException, RestJsonClient}
+
+
+class MLFRepositoryRestClient(hostUrl: URL) extends ModelRepository {
+  type RestResponse[RESPONSE] = Either[RestClientException, RESPONSE]
```

Comment:
ModelRepository also needs two layers I think, so let's say there shoudn't be any Rest word in these objects (but the rest objects should know how to map to itself to some instance of ModelRepository or Model)